src/components/Form.js

```diff
@@ -95,7 +99,7 @@ class Form extends React.Component {
 
         this.state = {
             errors: {},
-            inputValues: {},
+            inputValues: props.draftValues,
```

Comment:
This change in the PR caused https://github.com/Expensify/App/issues/14784 regression. 

Title: Settings - Changed name is not saved under Display name page

Steps to reprodcue:
- Click on >Setting>Profile > Display name
- Change name>Click Save
- Click the back arrow
- Click the Display name again
- The new name does not match the old name